src/local-settings.js

```diff
@@ -26,6 +26,7 @@ const CONFIGURATION_VARS = [
   ],
   [
     ENS_REGISTRY_ADDRESS,
+    getUrlParam('ens'),
```

Comment:
Shouldn't we give this a sensible default?

Reply:
What do you mean exactly?